### DISCUSSION TITLE
Editing colormap label after plotting a 2D dataset

### DISCUSSION THREAD

**Discussion by paniash** [pydata/xarray#9895] [Q&A] [answered]
Hi all,

I recently started playing around with xarray, and encountered the case where I'd like to change the labels of the axes for a 2D plot. Of course, for changing the x,y labels I simply use `ax.set_xlabel("x units")` and `ax.set_ylabel("y units")` methods. How do I do that for the colormap?

A minimal working example:
```python
import matplotlib.pyplot as plt

ax = plt.figure().gca()
y = ds["response_amplitude"]   # ds is my xarray dataset
y.plot(ax=ax)
ax.set_xlabel("some x label")
```

Not sure how I would access the colormap object here. I'd really appreciate any help.

**Comment by scottyhq** [comment, marked as answer]
@paniash there are various ways to do this, for example as shown in the docs here https://docs.xarray.dev/en/stable/user-guide/plotting.html#other-features

```python
y.plot(cbar_kwargs={"label": "my custom label"},
```

If you looked other places in the docs and expected to find this information, it'd be helpful to know.
